libafl/src/executors/timeout.rs

```diff
@@ -59,42 +60,29 @@ impl Debug for Timeval {
 }
 
 #[repr(C)]
-#[cfg(unix)]
+#[cfg(all(unix, target_vendor = "apple"))]
```

Comment:
`#[cfg(target_vendor = "apple")]` should suffice, I don't hope they every do a non-unix OS